ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-service-insertion.md

```diff
@@ -5,6 +5,10 @@
 - [Management](#management)
   - [Management Interfaces](#management-interfaces)
 - [Routing](#routing)
+- [Connections](#connections)
+  - [Connections Through Ethernet Interface](#connections-through-ethernet-interface)
+  - [Connections Through Tunnel Interface](#connections-through-tunnel-interface)
+- [Router Service-Insertion Configuration](#router-service-insertion-configuration)
```

Comment:
Hmm just noticed this - we will need a new section actually - let me fix the jinja template

Reply:
Committed all changes.